ci/build_container/build_container_centos.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# Note: rh-git218 is needed to run `git -C` in docs build process.
+yum install -y centos-release-scl epel-release
+yum update -y
+yum install -y devtoolset-7-gcc devtoolset-7-gcc-c++ devtoolset-7-binutils java-1.8.0-openjdk-headless \
+    rh-git218 wget unzip which make cmake3 patch ninja-build devtoolset-7-libatomic-devel openssl python27
+
+ln -s /usr/bin/cmake3 /usr/bin/cmake
+ln -s /usr/bin/ninja-build /usr/bin/ninja
+
+BAZEL_VERSION="$(curl -s https://api.github.com/repos/bazelbuild/bazel/releases/latest |
+    python -c "import json, sys; print json.load(sys.stdin)['tag_name']")"
+BAZEL_INSTALLER="bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"
+curl -OL "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/${BAZEL_INSTALLER}"
+chmod u+x "./${BAZEL_INSTALLER}"
+"./${BAZEL_INSTALLER}"
+rm "./${BAZEL_INSTALLER}"
+
+# Required by rh-git218
+echo "/opt/rh/httpd24/root/usr/lib64" > /etc/ld.so.conf.d/httpd24.conf
+ldconfig
+
+# CentOS toolchains doesn't have C++11 ABI so needs ignore, also punting PATH into system bazelrc.
+echo "build --copt=-DENVOY_IGNORE_GLIBCXX_USE_CXX11_ABI_ERROR=1 --action_env=PATH=/opt/rh/devtoolset-7/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" >> /etc/bazel.bazelrc
```

Comment:
Maybe just do `--action_env=PATH` since you're setting the PATH explicitly on the next line? Or even flip the two lines and do `--action_env=PATH=$PATH` if it's possible that build_container_common.sh modifies the PATH.